SergioMendozaParra/August 26/Challenge_Polymorphism_Inheritance_Encapsulation/Main.java

```diff
@@ -0,0 +1,12 @@
+package myPackage;
```

Comment:
Use meaningful package names

Reply:
Agree, I'm going to fix the package names and upload them again. Thanks.